caom2repo/setup.cfg

```diff
@@ -19,22 +19,22 @@ test = pytest
 omit = */tests/*
 
 [tool:pytest]
-minversion = 2.2
+minversion = 2.3
```

Comment:
I think this is the minversion of pytest and not caom2 library. Not sure what the original reason for having it was. It should probably be removed as the current pytest is at version 3.*